.circleci/lint.sh

```diff
@@ -18,23 +18,18 @@ MODULE="$(basename -s .info.yml -- ./*.info.yml)"
 
 #-------------------------------------------------------------------------------
 
-echo "==> Lint code for module $MODULE."
-echo "  > Running PHPCS."
-build/vendor/bin/phpcs \
-  -s \
-  -p \
-  --standard=Drupal,DrupalPractice \
-  --extensions=module,php,install,inc,test,info.yml,js \
-  "${BUILD_DIR}/web/modules/${MODULE}"
-
-echo "  > Running drupal-check."
-build/vendor/bin/drupal-check \
-  --drupal-root=build/web \
-  "${BUILD_DIR}/web/modules/${MODULE}"
+pushd "${BUILD_DIR}" >/dev/null || exit 1
+
+echo "==> Lint code."
+echo "  > Running PHPCS, PHPMD, TWIGCS"
+vendor/bin/phpcs
```

Comment:
please separate these to run each with own sub-title

Reply:
I have updated it.